app/templates/partials/summary/summary.html

```diff
@@ -2,8 +2,8 @@
     {%- if content.summary.collapsible -%}
         {%- include 'partials/summary/collapsible-summary.html' -%}
     {%- else -%}
-        {%- for group in content.summary.groups -%}
-            {%- if group.title -%}
+        {%- for group in content.summary.groups if group.blocks-%}
+            {%- if group.title and group.blocks-%}
```

Comment:
is ` and group.blocks` redundant as it cant get in here if no blocks?

Reply:
My bad.